### PR TITLE
Minor grammar & consistency updates

### PR DESCRIPTION
**Pull request by kaihoffman**
Change one sentence in the Ingress page to flow grammatically ("different ingress controller support" to "different ingress controllers support") and change "master" to "main" in CONTRIBUTING.md

### CODE REVIEW

**Comment by natalisucks** [comment]
Thanks for these great little tweaks, @kaihoffman! We appreciate your first contribution and look forward to many more 🎉 

Would you mind [squashing your commits](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits) before we merge? That would be really great.

Unofficial LGTM before we squash and merge 💪

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,6 +34,6 @@ Note that code issues should be filed against the main kubernetes repository, wh
 
 ### Submitting Documentation Pull Requests
 
-If you're fixing an issue in the existing documentation, you should submit a PR against the master branch.  Follow [these instructions to create a documentation pull request against the kubernetes.io repository](http://kubernetes.io/docs/home/contribute/create-pull-request/).
+If you're fixing an issue in the existing documentation, you should submit a PR against the main branch.  Follow [these instructions to create a documentation pull request against the kubernetes.io repository](http://kubernetes.io/docs/home/contribute/create-pull-request/).
```
Comment (bradtopol) [inline CONTRIBUTING.md:37]: Great job!

**Pull request merged**